dev/com.ibm.ws.threading/src/com/ibm/ws/threading/internal/ExecutorServiceImpl.java

```diff
@@ -278,6 +280,37 @@ private synchronized void createExecutor() {
         }
     }
 
+    private static ClassLoader getContextClassLoader(Thread thread) {
```

Comment:
I do wonder if you believe that these two new methods should be marked with `@Trivial`